editor/editor_inspector.cpp

```diff
@@ -627,10 +627,13 @@ void EditorProperty::gui_input(const Ref<InputEvent> &p_event) {
 						new_coords.x = 0;
 						new_coords.y++;
 					}
-
-					call_deferred(SNAME("emit_changed"), property, new_coords, "", false);
+					if (new_coords.x < object->get("hframes").operator int64_t() && new_coords.y < object->get("vframes").operator int64_t()) {
```

Comment:
Shouldn't we use `int64_t()` instead of `.operator int64_t()`? We generally prefer not using the `.operator` syntax when possible.

Reply:
Sure I agree, wasn't sure if we did that for a particular reason so I kept it consistent. I'll push a change for that

Reply:
> We generally prefer not using the .operator syntax when possible.

First time I hear this.

Reply:
Yeah that's not an established rule, I use both myself at different times. We could try to harmonize this but I'm not sure what are the pros and cons for each (I guess the `operator` syntax would fail compiling if that operator doesn't exist, which is better than a cast returning an invalid result... but for `Variant` we kind of know what casts are valid).